GDV-HERBST/appendix/LKF.md

```diff
@@ -341,10 +366,52 @@ $$
 
 ## Wie werden die Berechnungsverfahren wie Phong, Metal, Blinn usw. aus technischer Sicht bezeichnet?
 
+- TODO
+- Beleuchtungsmodell/Shading Modells
+- Nutzung Halfway-Vektoren
+
 ## Welche Bedeutung haben die grafischen Schnittstellen OpenGL, DirectX/3D?
 
+- Gemeinsamkeiten:
+  - CG-APIs zum Rendern von 2D und 3D CG.
+  - GPU haben normalerweise mindestens eine der beiden implementiert
+- **OpenGL**
+  - offener Standard (größtenteils)
+  - Cross-Platform
+  - OpenGL-ähnliche Bibliotheken für viele Unix-basierte Systeme
+    - Mac OS X, Linux, Nintendo, Sony
+  - Teilmenge von OpenGL als Haupt-Grafikbibliothek:
+    - Android, BlackBerry, iOS [..]
+- **DirectX / Direct3D**
+  - proprietäre Lizenz
+  - Microsoft $\rightarrow$ Windows, Xbox
+
+- TODO
+
+[Wikipedia](https://en.wikipedia.org/wiki/Comparison_of_OpenGL_and_Direct3D)
+
 ## Welche Bestandteile des JT-Datenformates können Sie benennen?
 
+> JT (Jupiter Tesselation) ist ein ISO-Standard-Grafikformat für 3D-Daten.
+
+Das Datenmodell unterstützt unterschiedlichste Repräsentationen der CAD-Geometrie:
+
+- „tesselierte“ Dreiecksflächen-Geometrie
+- „exakte“ Brep-Geometrie: JT-BREPs; XT-BREPs; LIBRA-Format (stark komprimieret, verlustbehaftetet)
+
+**Szenengraph** mit CAD-spezifischen Knoten und Attribute-Unterstützung
+
+- Facetteninformation (**Dreiecke**) ausgereiften Kompressionstechniken.
+- mathematisch **exakte Geometriedefinition**
+- Produktstruktur, **Produkt- und Fertigungsinformation** (PMI),
+- **visuelle Attribute** wie Licht, Textur und Shader
+- **Metadaten**
+
 ## Worum handelt es sich bei der Abkürzung "BREP"?
 
+> **B**oundary **Rep**resentation, ist eine Darstellungsform eines Flächen- oder Volumenmodells, in der Objekte durch ihre begrenzenden Oberflächen beschrieben werden.
+
 ## Probleme geometrische Schnittstellen/Direktschnittstellen/Neutralformate
+
+- TODO
+- HÄ? Verluste, Ungenauigkeiten in Neutralformaten?
```

Comment:
Ja ich glaube auch; manche Dinge können in manchen Formaten dann nicht richtig gespeichert werden, vorallem irgendwelche proprietären Sachen von einem speziellen Programm. Bei Neutralformaten, die nur tesselierte Modelle speichern, geht die BREP natürlich verloren.